datacube/cfg/api.py

```diff
@@ -122,8 +122,8 @@ def __init__(
 
     @classmethod
     def get_environment(cls,
-                        env: "ODCEnvironment" | str | None = None,
-                        config: "ODCConfig" | str | PathLike | list[str | PathLike] | None = None,
+                        env: Union["ODCEnvironment", str, None] = None,
+                        config: Union["ODCConfig", str, PathLike, list[str | PathLike], None] = None,
```

Comment:
This is getting long enough to justify having a type alias defined, I would then move comment about needing `Union` instead `|` next to that alias rather than import statement.